terraform/modules/rds/rds.tf

```diff
@@ -0,0 +1,49 @@
+resource "aws_db_subnet_group" "ccDBSubnetGroup" {
+  name = "cc-db-subnet-group"
+  subnet_ids = [
+    var.cc_private_subnets[0].id,
+    var.cc_private_subnets[1].id
+  ]
+  tags = {
+    Name    = "ccDBSubnetGroup"
+    Project = "CC TF Demo"
+  }
+}
+
+resource "aws_security_group" "ccDBSecurityGroup" {
```

Comment:
Can we allow inbound ssh traffic from any ip? Also do you not need inbound rules for every service we plan to connect? 
TCP 9200, TCP 6379, TCP 3600, TCP 27017